docs/topics/install/docker.rst

```diff
@@ -140,6 +135,81 @@ migrations::
 Gotchas!
 ~~~~~~~~
 
+Here's a list of a few of the issues you might face when using docker.
+
+Can't access the web server?
+----------------------------
+
+Check you've created an hosts file entry pointing ``olympia.dev`` to the
+relevant ip address.
+
+Also make sure you've copied ``docker-compose.override.yml.tmpl`` to
```

Comment:
Yeah, I’d go for the easiest defaults first; if they have to change
something (which I’d venture most shouldn’t have to) it’s alright if it’s
more work as long as we explain it :-)